src/collective/fingerpointing/subscribers/genericsetup_logger.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""Event subscriber handler for IProfileImportedEvent."""
+from collective.fingerpointing.config import AUDIT_MESSAGE
+from collective.fingerpointing.interfaces import IFingerPointingSettings
+from collective.fingerpointing.logger import log_info
+from collective.fingerpointing.utils import get_request_information
+from plone import api
+
+
+def profile_imports_logger(event):
+    """Log genericsetup profile imports."""
+    record = IFingerPointingSettings.__identifier__ + '.audit_profile_imports'
+    audit_profile_imports = api.portal.get_registry_record(record, default=False)  # noqa: E501
```

Comment:
a better style is to write this multiline instead of a noqa:
```
audit_profile_imports = api.portal.get_registry_record(
    record, 
    default=False,
)
```